homeassistant/helpers/entity.py

```diff
@@ -551,6 +551,19 @@ def __repr__(self) -> str:
         """Return the representation."""
         return "<Entity {}: {}>".format(self.name, self.state)
 
+    # call an requests
+    async def async_request_call(self, coro, *args, **kwargs):
```

Comment:
So I suggest that you actually pass in a coroutine. This currently is a coroutine function that produces a coroutine. Since coroutines are only executed once they are awaited, it's fine to have them pass in as a coroutine. 

```python
await light.async_request_call(light.async_turn_off(**off_pars))
```

Reply:
Yeah is better